### PR TITLE
Motor B fix 1

### PR DESCRIPTION
**Pull request by berkserbet**
### Why:
* Contains fix for lowering motor b value. This method divides the heating, cooling and ventilation values by three before the sensitivity analysis step. There is another fix in the works that will be compared.

### What:
* Motor B values have been slightly lowered. 

### Checklist
- [ ] Finished My Changes
- [ ] Automated Unit Tests

### CODE REVIEW

File: pipelines/ceus/tasks/find_sensitivities.py [hunk 1]
```diff
@@ -92,19 +92,22 @@ def _task(self):
 
                     if enduse == 'Heating':
                         A_heat = A[:, :-1]
-                        x = self.get_baseload(enduse_df, A_heat)
+                        x = self.get_baseload(enduse_df, A_heat)/3
```
Comment (jongoncalves) [inline pipelines/ceus/tasks/find_sensitivities.py]: Spaces around `/`

File: pipelines/ceus/tasks/find_sensitivities.py [hunk 2]
```diff
@@ -92,19 +92,22 @@ def _task(self):
...
                         x = np.append(x, [0])
 
                     elif enduse == 'Cooling':
                         A_cool = np.delete(A, -2, 1)
-                        x = self.get_baseload(enduse_df, A_cool)
+                        x = self.get_baseload(enduse_df, A_cool)/3
```
Comment (jongoncalves) [inline pipelines/ceus/tasks/find_sensitivities.py]: same

File: pipelines/ceus/tasks/find_sensitivities.py [hunk 3]
```diff
@@ -92,19 +92,22 @@ def _task(self):
...
                         cooling_sensitivity = x[-1]
                         x = np.append(x[:-1], [0])
                         x = np.append(x, [cooling_sensitivity])
 
                     else:
                         A_base = A[:, :-2]
-                        x = self.get_baseload(enduse_df, A_base)
+                        if enduse == 'Ventilation':
+                             x = self.get_baseload(enduse_df, A_base)/3
```
Comment (jongoncalves) [inline pipelines/ceus/tasks/find_sensitivities.py]: same

File: pipelines/rbsa/tasks/find_sensitivities.py [hunk 4]
```diff
@@ -99,19 +99,22 @@ def _task(self):
 
                 if enduse == 'Heating':
                     A_heat = A[:, :-1]
-                    x = self.get_baseload(enduse_df, A_heat)
+                    x = self.get_baseload(enduse_df, A_heat)/3
```
Comment (jongoncalves) [inline pipelines/rbsa/tasks/find_sensitivities.py]: same

File: pipelines/rbsa/tasks/find_sensitivities.py [hunk 5]
```diff
@@ -99,19 +99,22 @@ def _task(self):
...
                     x = np.append(x, [0])
 
                 elif enduse == 'Cooling':
                     A_cool = np.delete(A, -2, 1)
-                    x = self.get_baseload(enduse_df, A_cool)
+                    x = self.get_baseload(enduse_df, A_cool)/3
```
Comment (jongoncalves) [inline pipelines/rbsa/tasks/find_sensitivities.py]: same

File: pipelines/rbsa/tasks/find_sensitivities.py [hunk 6]
```diff
@@ -99,19 +99,22 @@ def _task(self):
...
                     cooling_sensitivity = x[-1]
                     x = np.append(x[:-1], [0])
                     x = np.append(x, [cooling_sensitivity])
 
                 else:
                     A_base = A[:, :-2]
-                    x = self.get_baseload(enduse_df, A_base)
+                    if enduse == 'Ventilation':
+                         x = self.get_baseload(enduse_df, A_base)/3
```
Comment (jongoncalves) [inline pipelines/rbsa/tasks/find_sensitivities.py]: same

File: pipelines/ceus/tasks/find_sensitivities.py (hunk 1 above)
Comment (berkserbet) [reply pipelines/ceus/tasks/find_sensitivities.py]: fixed

**Pull request merged**